dask/tests/test_base.py

```diff
@@ -168,6 +171,15 @@ def test_tokenize_numpy_ufunc_consistent():
     assert tokenize(inc) == tokenize(inc)
 
 
+def test_tokenize_partial_func_args_kwargs_consistent():
+    f = tz.partial(f3, f2, c=f1)
+    res = normalize_token(f)
+    sol = (b'cdask.tests.test_base\nf3\np0\n.',
+           (b'cdask.tests.test_base\nf2\np0\n.',),
+           (('c', b'cdask.tests.test_base\nf1\np0\n.'),))
```

Comment:
This test seems somewhat fragile to future changes

Reply:
No more so than the other tests checking explicit `tokenize` results. Couldn't think of a better way to check that args/kwargs were tokenized. I think this fine as is.